templates/support/OAuthAccessTokenManager.swift.hbs

```diff
@@ -0,0 +1,126 @@
+//  
+//  {{>frag/generatedBy}}
+//
+
+/** A token manager that will group refresh attempts to prevent concurrent token requests */
+import Foundation
+
+actor OAuthAccessTokenManager {
+    
+    var accessToken: OAuthToken? {
```

Comment:
Perhaps this type should be `OAuthAccessToken`... I think "access token" is the right terminology for this thing.